src/app/routes/enrolment/enrolment-list/services/enrolment-filter-list.service.spec.ts

```diff
@@ -57,13 +62,13 @@ describe('EnrolmentFilterListService', () => {
     });
   });
 
-  it('should filter by namespace', (done) => {
+  it('should filter by expired claims', (done) => {
```

Comment:
filter by namespace doesn't matter anymore? I do not know why you're changing existing test instead of adding new.

Reply:
I just accidentally updating it rather than copying it to change.